sunpy/data/data_manager/manager.py

```diff
@@ -158,28 +177,6 @@ def skip_hash_check(self):
         finally:
             self._skip_hash_check = False
 
-    def get(self, name):
```

Comment:
This is not needed?

Reply:
I have made an internal functon _download_and_cache_file which handles so the get function is defined over  https://github.com/sunpy/sunpy/pull/7845/files#diff-a866802ae6c134393492683556a9de3e34fb7d54c8e2c4f604d288094c30a55dR111-R137